src/Uno.UI/UI/Xaml/Shapes/Shape.cs

```diff
@@ -104,6 +104,7 @@ public double StrokeThickness
 			typeof(Shape),
 			new FrameworkPropertyMetadata(
 				defaultValue: 1.0d,
+				FrameworkPropertyMetadataOptions.AffectsMeasure,
```

Comment:
This was intentionally not added. In `OnStrokeThicknessChanged`, we only invalidate measure if stroke is not null.

Reply:
If we want to follow what WinUI is doing here, we should then delete the propertyChangedCallback.

@dr1rrb What are your thoughts?

Reply:
Didn't catch that, good point! Can revert based on what you think

Reply:
I have no strong opinion as long as we keep one of `AffectsMeasure` and `OnStrokeThicknessChanged`. Will let @dr1rrb weigh on which one we should use.

Reply:
Yes I think that yes we should not cause an invalidate measure when we can prevent them like in this case where the `Stroke is null` ... even if cleaner to just have the flag on all properties, it will have a negative perf impact so I would keep the `OnStrokeThicknessChanged` callback.